src/components/roles/RoleForm.tsx

```
@@ -0,0 +1,92 @@
+      id: initialRole?.id ?? 0,
+      name,
+      description,
+      permissions,
+      active: true,
+    });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="mb-4">
+        <label className="block text-gray-700 mb-1">Role name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)